packages/frames.js/src/middleware/images-worker/next/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./createImagesWorker";
```

Comment:
same here, this file could contain the code of `createImagesWorker.tsx`

Reply:
oh okay i see what you mean. will update